js/src/inline-links/inline.js

```diff
@@ -0,0 +1,508 @@
+import PropTypes from "prop-types";
```

Comment:
This file seems taken directly from Gutenberg, albeit at an older version than the latest.

Probably not something for this PR but we should ideally try to figure out a different approach as this will be hell to keep up-to-date and is bound to lead to issues where we start introducing regressions into Gutenberg as they've fixed something in their format-library but due to us replacing our own already older version reintroducing those issues.

Be it by using inheritance so we can only overwrite the funtions we need to, creating patches and dynamically creating these files during our build process or creating a PR with Gutenberg to do this in a better way.